### DISCUSSION TITLE
Compilation error when trying to import https in axios boot file of my Quasar App

### DISCUSSION THREAD

**Discussion by Ammamon** [quasarframework/quasar#11149] [General - Components / Directives / etc]
In my quasar app, I was trying to configure my axios instance for ssl connection, but getting compilation errors as shown below. Please help. I had tried to install https also, but same error occurring.

My code snippet:
```
import { boot } from 'quasar/wrappers';
import axios, { AxiosInstance } from 'axios';
import https from 'https';
...

const api = axios.create({baseURL: 'https://localhost:8443/asset', httpsAgent: new https.Agent({rejectUnauthorized: false}), });
...

```

Error log at console:

![image](https://user-images.githubusercontent.com/7417777/139295480-e4de86ab-8e94-47ca-acd1-44dd15dad374.png)

Error log on browser:

![image](https://user-images.githubusercontent.com/7417777/139295748-a111fb16-876e-4070-83f8-62d2adab1fe3.png)

**Comment by pdanpdan** [comment]
Maybe you don't have `https` installed? Btw, what is it?

Check here, maybe it helps: https://github.com/ethers-io/ethers.js/issues/998

**Reply by Ammamon** [reply]
I have tried installing both https and https-browserify.  But don't know how to configure https alias of the latter in quasar..conf.js.

The backend is secured for https access and I have successfully tested it using Postman by disabling its SSL certificate verification.

I am trying to do the similar thing via axios in my quasar app

**Reply by hawkeye64** [reply]
You could use the `proxy` directive in quasar.conf.js to redirect to your server.

**Comment by slavasgod** [comment]
@Ammamon Did you find any solution?
